schema/table.go

```diff
@@ -159,21 +165,20 @@ func (t Table) TableNames() []string {
 }
 
 // Call the table resolver with with all of it's relation for every reolved resource
-func (t Table) Resolve(ctx context.Context, meta ClientMeta, parent *Resource, resolvedResources chan<- *Resource) SyncSummary {
+func (t Table) Resolve(ctx context.Context, meta ClientMeta, parent *Resource, resolvedResources chan<- *Resource) (summary *SyncSummary) {
 	tableStartTime := time.Now()
-	summary := SyncSummary{}
-	meta.Logger().Info().Str("table", t.Name).Msg("fetch start")
+	meta.Logger().Info().Str("table", t.Name).Msg("table resolver started")
 
 	res := make(chan interface{})
 	startTime := time.Now()
 	go func() {
 		defer func() {
 			if err := recover(); err != nil {
+				stack := fmt.Sprintf("%s\n%s", err, string(debug.Stack()))
 				sentry.WithScope(func(scope *sentry.Scope) {
 					scope.SetTag("table", t.Name)
-					sentry.CurrentHub().Recover(err)
+					sentry.CurrentHub().CaptureMessage(stack)
```

Comment:
Sentry should already include the stack trace?

Reply:
their logic is incorrect it doesn't print it in the right way and try to do some "smart things" - just a string of the stack trace should work for us I think.